docs/getting_started/tutorial01.rst

```diff
@@ -13,13 +13,14 @@ Logging in
 Navigate to http://localhost:8000/admin/ and log in with the username
 and password you created with the ``python manage.py createsuperuser`` command.
 
-After logging in, you will be taken to the admin dashboard, also known as the "Wagtail Admin".
+After logging in, you will be taken to the admin dashboard - also known as the "Wagtail Admin".
 
 .. figure:: img/tutorial_admin.png
     :alt: The admin dashboard.
 
     The admin dashboard.
 
+.. _site-name:
```

Comment:
Question: what is this directive do?

Reply:
This makes a reference link to the section it's above that I used in the customize_design.rst file.  https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role